views/index.erb

```diff
@@ -1,27 +1,43 @@
-<h2>
-  <%= flash[:notice] %>
-</h2>
-Chitter
-<% if @user %>
-  <h1>Welcome, <%=@user.first['name'] %></h1>
-  <form action="/signout" method="post">
-    <input type="submit" value="Sign out" name="signout">
-  </form>
-  <p>
-  <form action='/peep' method='post'>
-    <label for='peep_text'>Peep:</label>
-    <input type='text' name='peep_text'>
-    <input type='submit' value='Post' name='post'>
-  </form>
-<% else %>
-  <a href='/sessions/new' class='clickable-text'>Sign in</a>
-  <a href='/users/new' class='clickable-text'>Create Account</a>
-<% end %>
-
-<ul>
-<% Peeps.all.each do |peep| %>
-<li class = "peep" >
-  <%=peep.date %> <%=peep.time %><br>
-  <%=peep.name%> peeped: <%=peep.text %><p>
-<% end %>
-</ul>
+<link rel="stylesheet" href="/style.css">
```

Comment:
If you want to link your stylesheet to all files without having to go through every single file, you can add it to a layout.erb, which uses yield in the body. That means that every single page has got the same stylesheet - that way you only need to specify a stylesheet in each erb file if it uses a different one!

Reply:
Thanks for this tip :D